types/express-serve-static-core/index.d.ts

```diff
@@ -961,7 +961,7 @@ export interface Response<
      * After middleware.init executed, Response will contain req property
      * See: express/lib/middleware/init.js
      */
-    req?: Request;
+    req: Request;
```

Comment:
this is a change for express-serve-static-core. did you want this included in this PR?

Reply:
This was done in 15.9 (also yes).